assets/schemas/target_genomes.json

```diff
@@ -30,6 +30,18 @@
           }
         ]
       },
+      "vcf_genotype_field": {
+        "description": "Specify whether to import genotypes (default: GT), or imputed dosages (DS) from the VCF file.",
+        "anyOf": [
+          {
+            "type": "string",
+            "pattern": "/^(GT|DS)$/"
+          },
+          {
+            "type": "null"
+          }
+        ]
+      },
```

Comment:
Testing this new schema against the example in `assets` fails validation. I'll double check and push any changes to this branch

Reply:
the schema is OK to represent the samplesheet, I'll make a target genomes schema to match the output of `samplesheet_to_json` but it's not urgent